evap/evaluation/templates/faq.html

```diff
@@ -14,7 +14,7 @@ <h3 class="mb-4">{% trans "These are the Frequently Asked Questions (FAQs)" %}</
         {% for section in sections %}
             <h4 id="{{ section.id }}-s">{{ section.title }}</h4>
             {% for question in section.questions.all %}
-                <div class="card{% if not forloop.last %} mb-1{% else %} mb-4{% endif %}">
+                <div id="card-{{ question.id }}" class="card{% if not forloop.last %} mb-1{% else %} mb-4{% endif %}">
```

Comment:
how about we call this `{{ question.id }}-card` or something like that and rename the div with the answer `{{ question.id }}-a` to be more consistent??